index.html

```diff
@@ -16,28 +16,51 @@ <h2>Try it:</h2>
       <textarea maxlength="1024" id="input-text" cols="60" rows="10">aeiou</textarea>
       <br>
       <button type="submit">Play</button>
+      <a href="javascript:stopPlayback()"><button type="button">Stop</button></a>
+      <a href="javascript:exportAudio()"><button type="button">Export</button></a>
     </form>
     <audio id="result-audio">
       Your browser does not support the HTML5 audio element.
     </audio>
     <script type="text/javascript">
       let lastText = null;
       let audio = null;
+      let playing = false;
       function tts() {
         let text = document.getElementById('input-text').value;
 
-        if (text !== lastText) {
+        if (text !== lastText) {` +
```

Comment:
I'm not sure exactly what happened here, but you have a stray `` ` + ``

Reply:
that's a leftover from some trickery I used to change the website for me in a userscript. removed in the latest commit to my branch